.gitignore

```diff
@@ -0,0 +1 @@
+/file::memory:?cache=shared
```

Comment:
this is not correct, needs to be a real file ... already saw this in another repo
it's supposed to be in-memory but that broke

Reply:
Yeah, it's weird. But it created a file on my machine 🤷🏼

Reply:
see https://github.com/grosser/ar_multi_threaded_transactional_tests/blob/master/test/database.rb

Reply:
I have removed the .gitignore file. But I am not sure how to fix sqlite. Is it sqlite or the sqlite adapter that is broken?